dart/lib/src/protocol/message.dart

```diff
@@ -0,0 +1,38 @@
+class Message {
```

Comment:
can we add a comment with a json example like all the other protocols?
eg https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/protocol/breadcrumb.dart#L6-L19